Tests/FluentAssertions.Json.Shared.Specs/JTokenAssertionsSpecs.cs

```diff
@@ -457,6 +457,70 @@ public void When_jtoken_does_have_value_NotHaveValue_should_fail()
 
         #endregion (Not)HaveValue
 
+        #region (Not)MatchRegex
+
+        [Fact]
+        public void When_json_matches_regex_pattern_MatchRegex_should_succeed()
+        {
```

Comment:
❌ You're still using the word `MatchRegex`....

Reply:
Braces done.

About the `MatchRegex`, they are almost **all** like that:

`When_objects_are_equal_NotBeEquivalentTo_should_fail`
`When_only_the_order_of_properties_differ_BeEquivalentTo_should_succeed`
`When_both_objects_are_null_BeEquivalentTo_should_succeed`
`When_expecting_the_actual_number_of_elements_HaveCount_should_succeed`
`When_jtoken_is_an_empty_object_ContainSingleItem_should_fail`
etc.

So I strongly feel that consistency is better for now and doing an overhaul pass to fix them all in another commit is the way to go. It's 🤡 all the way ;)